### DISCUSSION TITLE
There was an error running Lighthouse: `DNS servers could not resolve the provided domain.` (code: DNS_FAILURE)

### DISCUSSION THREAD

**Discussion by Achchheweb** [spatie/lighthouse-php#29] [Q&A]
Hi i have setup every thing as you mentioned in the docs. 
It's working fine on Local Serve for every sites.
The issue is when i make it live or move it server with all configurations the some site having issue with **403** and **DNS_FAILURE** 
for some sites only otherwise it working properly i have also set the **extraHeaders** after that some site's issue resolve but still remaining site's lighthouse reports is not working. Kindly suggest me to how to get the results for these sites.

https://unifiedportal-emp.epfindia.gov.in/epfo/
![image](https://github.com/spatie/lighthouse-php/assets/141727306/c2104548-b93b-4046-a33a-fd04d040876c)

https://www.justdial.com/

![image](https://github.com/spatie/lighthouse-php/assets/141727306/4395c238-4ecf-4abc-acc7-3606bcb0e423)

with other sites
![image](https://github.com/spatie/lighthouse-php/assets/141727306/07e3c2af-f33b-43c8-922d-79b6be0a07d7)

**Comment by henrich45** [comment]
Well, I have used this website and you are right about it that it works slow, so I switched to [epfo: नए बदलावों का उद्देश्य सिस्टम को यूज़र-फ़्रेंडली और तेज़ बनाना है](https://theepfoportal.com/) as it delivers faster results in less time. It's interface is also quite helpful and provide very valuable information related the EPFO.
